stake-pool/py/stake/state.py

```diff
@@ -48,24 +48,24 @@ class StakeAuthorize(IntEnum):
     WITHDRAWER = 1
 
 
-class StakeStateType(IntEnum):
+class StakeStateV2Type(IntEnum):
     """Stake State Types."""
     UNINITIALIZED = 0
     INITIALIZED = 1
     STAKE = 2
     REWARDS_POOL = 3
 
 
-class StakeState(NamedTuple):
-    state_type: StakeStateType
+class StakeStateV2(NamedTuple):
+    state_type: StakeStateV2Type
```

Comment:
assuming this is also accidental because nothing about the definitions has changed at all...

Reply:
ok yes im sure about this because flags are not added to `STAKE_STATE_LAYOUT`/`STAKE_AND_META_LAYOUT` nor were they renamed

Reply:
You're right, this is also incorrect